terraform/README.md

```diff
@@ -60,8 +60,38 @@ For these commands ensure you are in the terraform initialised folder when runni
 ## How it works
 #### main.tf
 - This contains the terraform config required for provisioning the cloud architecture.
```

Comment:
Clear explanation of what exactly is being built, and a high-level overview of how exactly the main.tf file does this; like for the iterative generation of Lambdas for example.